wooCommerce_scripts/Scripts/WooCommerceUtility.py

```diff
@@ -22,6 +23,22 @@ def assert_status_code(self):
         assert self.status_code == self.expected_status_code, f"if bad status code."\
           f"Expected {self.expected_status_code}, Actual status code: {self.status_code},"\
           f"URL: {self.url}, Response Json: {self.rs_json}"
```

Comment:
add a space after the function. between two functions there should be 1 empty line.

Reply:
Looks like you are doing the same in almost all functions so modify all and add spaces.